docs/scos/dev/tutorials-and-howtos/howtos/howto-delete-store-with-ds.md

```diff
@@ -0,0 +1,172 @@
+---
+title: "HowTo: Delete store when using the dynamic store feature"
+description: This document shows how to delete store when the system is running with a dynamic store feature.
+past_updated: Jun 26, 2023
+template: howto-guide-template
+---
+
+This document describes the steps to consider how manually delete store in spryker when using dynamic store.
+
+## When it might be useful
+
+Often, the store that was created earlier becomes unnecessary. It can be a store that has been used for a brand, product, holiday, or marketing promotions or events аnd ending experiments with no production environment. 
+
+On the technical side, removing such a store helps:
+1. Increasing the speed of the store and deleting redundant data.
+2. Reduction of memory and storage footprint.
+
+
+## Step-by-step plan
+
+**To check all your actions, it is better to perform all actions in a test environment. And also provide a backup of the database and other storages if necessary.**
+
+### 1. Backup Database.
+
+Before starting the process, you need to back up the database and check the possibility of recovery from it.
+If you are using AWS RDS, follow by link [Create and restore database backups](/docs/cloud/dev/spryker-cloud-commerce-os/create-and-restore-database-backups.html). 
+
+
+### 2. Suspend P&S.
+
+**In order to reduce the risks of data synchronization errors, you can suspend data synchronization through P&S.**
+
+1. Enable **maintenance mode** by using command: 
+```bash
+vendor/bin/console maintenance mode
+```
+Or if you are using AWS infrastructure please check documentation [Enable and disable maintenance mode](docs/cloud/dev/spryker-cloud-commerce-os/manage-maintenance-mode/enable-and-disable-maintenance-mode.html)
+
+2. Make sure there are not too many messages in the RabbitMQ. Wait for the messages to be processed.
```

Comment:
```suggestion
2. Make sure there aren't too many many messages showing in the RabbitMQ. Wait for the messages to be processed.
```
Editor's note: might be useful to explain what will /would happen if too many msgs are showing up.